uname: change sysname flag to kernel-name

tsliang opened the pull request:
The --sysname flag in GNU uname was deprecated in 2002 and replaced by --kernel-name.

Arcterus commented:
AFAICT `--sysname` is still supported, it's just hidden in the usage information.  Can you switch the util from `getopts` to `clap` and add `--sysname` as an alias?  That way the behavior should be the same as the GNU version.

tsliang commented:
Sure, will do. Should I be worried about that failed CI build btw, or does that happen to every PR?

Arcterus commented:
I don’t think it’s a problem.  The build just errored out because it couldn’t download the compiler (lol).

tsliang commented:
@Arcterus ok, everything's updated. I added the `--sysname` alias, along with `--release` (another obsolete alias). I also updated the option help list; it's now word-for-word the same with GNU uname.

Arcterus commented:
Do you mind changing the help menu back to what it was before?  I'm not sure if making it the exact same as the GNU version violates the GPL or whatever GNU licenses its documentation under.  Other than that this looks ready to go!

tsliang commented:
@Arcterus sorry for the late reply - holidays and all 😃 help menu has been changed to what it was before!

Arcterus commented:
Can you actually squash the last two?  Just to get rid of the GNU help strings from the commit history.

tsliang commented:
@Arcterus you got it

Arcterus commented:
Alright, thanks!  I'll merge this after one of the builds finishes.

Arcterus commented:
Also I just saw your comment about the holidays.  No worries, I've been busy too 😄

Arcterus commented on src/uname/uname.rs:
```diff
@@ -32,48 +47,76 @@ static HOST_OS: &'static str = "Darwin";
+            .short("s")
+            .long(OPT_KERNELNAME)
+            .alias("sysname") // Obsolescent option in GNU uname
+            .help("print the operating system name.")
```
I think you deleted a `)` by accident when you rebased.

tsliang replied:
Eep! Sorry about that.

Arcterus commented:
Alright seems good now.  Thanks!
